JavaScript/d-array-concat.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const benchmark = require('./2-benchmark.js');
+
+
+function testConcat() {
+  let arr1 = [1, 2, 3],  arr2 = [4, 5, 6];
+  arr1 = arr1.concat(arr2);
+  return arr1;
+}
+
+
+function testPushApply() {
+  let arr1 = [1, 2, 3],  arr2 = [4, 5, 6];
+  [].push.apply(arr1, arr2);
+  return arr1;
+}
+
```

Comment:
Can you please use a consistent number of empty lines between functions? Stick with either one or two, don't mix both.